applications/crossbar/src/modules_v1/cb_users_v1.erl

```diff
@@ -509,7 +509,10 @@ load_user(UserId, Context) -> crossbar_doc:load(UserId, Context, ?TYPE_CHECK_OPT
 %%------------------------------------------------------------------------------
 -spec validate_request(kz_term:api_binary(), cb_context:context()) -> cb_context:context().
 validate_request(UserId, Context) ->
-    prepare_username(UserId, Context).
+    Validators = [{fun is_username_min_length/2, []}
```

Comment:
`_v1` modules are considered deprecated. These edits should be in `_v2` by default, `_v1` if you really want them there.

Reply:
I'll address this with updated solution too